tests/python_build/fixtures/rd_glsl/_included.h

```diff
@@ -0,0 +1 @@
+#define M_PI 3.14159265359
```

Comment:
You'll want to rename this to ``_included.glsl`` to avoid those pesky CI errors. You'll need to do the same with the other shader include file.

Reply:
pipeline fixed, thanks for catching that